source/query-bar.txt

```diff
@@ -8,7 +8,7 @@ From the various Collections tabs, you can use the query bar to specify
 MongoDB queries.  For details, see :ref:`querying`.
 
 .. figure:: /images/compass/query-bar-documents-view.png
-   :figwidth: 780px
+   :figwidth: 805px
 
 You can specify a query filter in the query bar. If you click the 
```

Comment:
Good suggestion.  Added links to aggregation operators.

Previous two comments do not have a reply button, so including responses here:
* inclusive/exclusive: I think you're confusing 'limit' with "range".  This is a single integer, not a series/range, so the answer is implicitly inclusive.  Note that we don't bother to point this out even on the reference page that I've now linked to.
* define default: I'm going to invoke writer's privilege here and move on.